Crane/app/build.gradle

```
@@ -29,6 +29,11 @@ if (rootProject.file("local.properties").exists()) {
     properties.load(rootProject.file("local.properties").newDataInputStream())
 }
 
+// For testing on CI, a key can also be provided via an environment variable.
+// The local.properties key takes precedence
+var systemGoogleMapsKey = System.getenv("GOOGLE_MAPS_KEY") ?: ""
```

```suggestion
String systemGoogleMapsKey = System.getenv("GOOGLE_MAPS_KEY") ?: ""
```

Is this the right way to define these variables? Looking at the lines above, the type can be indicated? Also, I've seen `def` in other places